### ISSUE TITLE
iPad Multitasking Support

### ISSUE THREAD

**Issue by bennokress** [twostraws/CodeScanner#101]
In its current state the camera feed just freezes when using the Scanner on an iPad and activating one of the Multitasking Modes. [With iPadOS 16 Apple introduced support for having the camera active in Multitasking scenarios](https://developer.apple.com/documentation/avkit/accessing_the_camera_while_multitasking_on_ipad) and I think we should add that as an option.

I'll probably look into adding it in the coming days and open a PR if I have it in a working state.

**Comment by bennokress** [comment]
Sadly I'm not able to get this to work so far. If anyone has a way to improve this, I think it would improve the quality of this package though. For my personal use case I'm working around it for now by displaying a placeholder and a hint when the user enters a Multitasking Mode.

**Comment by kelvinjou** [comment]
@bennokress  Do you have code that I could test out? I'm going to attempt to add multitasking support next week.

**Comment by bennokress** [comment]
@kelvinjou No, I discarded my attempts and it's too long ago for me to remember what I did, sorry.
